turbot/table_turbot_mod_version.go

```diff
@@ -0,0 +1,138 @@
+	return &plugin.Table{
+		Name:        "turbot_mod_version",
+		Description: "Module versions in turbot organization.",
+		List: &plugin.ListConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "name", Require: plugin.Optional},
+				{Name: "status", Require: plugin.Optional},
+				{Name: "filter", Require: plugin.Optional},
+				{Name: "org_name", Require: plugin.Optional},
+			},
+			Hydrate: listModVersion,
+		},
+		Columns: []*plugin.Column{
+			// Top columns
+			{Name: "name", Type: proto.ColumnType_STRING, Description: "The name of the mod."},
+			{Name: "identity_name", Type: proto.ColumnType_STRING, Description: "The indentity name of the mod."},
+			{Name: "org_name", Type: proto.ColumnType_STRING, Transform: transform.FromQual("org_name"), Description: "The name of the organization."},
+			{Name: "status", Type: proto.ColumnType_STRING, Description: "The status of the mod version."},
+			{Name: "version", Type: proto.ColumnType_STRING, Description: "The version of the mod."},
+			{Name: "filter", Type: proto.ColumnType_STRING, Transform: transform.FromQual("filter"), Description: "Search text used for this mod list."},
+			{Name: "mod_peer_dependency", Type: proto.ColumnType_JSON, Transform: transform.FromField("Head.PeerDependencies"), Description: "Peer dependencies of the mod."},
+			// Other columns
+			{Name: "workspace", Type: proto.ColumnType_STRING, Hydrate: plugin.HydrateFunc(getTurbotWorkspace).WithCache(), Transform: transform.FromValue(), Description: "Specifies the workspace URL."},
+		},
+	}
+}
+
+type ModVersionInfo struct {
+	IdentityName string
+	Name         string
+	Status       string
+	Version      string
+	Head         ModVersionHead
+}
+
+const (
+	queryModVersions = `
+query modVersionSearchByName($search: String, $modName: String, $orgName: String, $status: [ModVersionStatus!]) {
+  modVersionSearches(search: $search, modName: $modName, orgName: $orgName, status: $status) {
+	items {
+	identityName
+	name
+	versions {
+		version
+		status
+		head
+	}
+	}
+	paging {
+	next
+	}
+  }
+}
+`
+)
+
+func listModVersion(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	conn, err := connect(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("turbot_mod_version.listModVersion", "connection_error", err)
+		return nil, err
+	}
+
+	var status interface{}
+	var modName, searchText, orgName string
+
+	quals := d.KeyColumnQuals
+
+	// Additional filters
+	if quals["status"] != nil {
+		status = strings.ToUpper(quals["status"].GetStringValue())
+	}
+	if quals["name"] != nil {
+		modName = quals["name"].GetStringValue()
+	}
+	if quals["filter"] != nil {
+		searchText = quals["filter"].GetStringValue()
+	}
+	if quals["org_name"] != nil {
+		orgName = quals["org_name"].GetStringValue()
+	}
+
+	// Default to a very large page size. Page sizes earlier in the filter string
+	// win, so this is only used as a fallback.
+	pageResults := false
```

Comment:
Did we miss adding this section of code - https://github.com/turbot/steampipe-plugin-turbot/blob/main/turbot/table_turbot_control.go#L136 or was it omitted intentionally?

Reply:
This is intentionally, this block code is for those GraphQL query which support pagination by passing `limit` with filter key.